sriov/common/utils.py

```diff
@@ -223,9 +223,9 @@ def vfs_created(ssh_obj, pf_interface, num_vfs, timeout = 10):
     cmd = "ls -d /sys/class/net/" + pf_interface + "v* | wc -w"
     print(cmd)
     for i in range(timeout):
+        time.sleep(1)     
```

Comment:
this line should already be on the main, did you rebase?

Reply:
@dkosteck Did you rebase to solve the conflict?

Reply:
@jianzzha Not yet, will push an update shortly